src/main/features/response/views/eligibility/under-18.njk

```diff
@@ -0,0 +1,27 @@
+{% extends "layout.njk" %}
+{% from "externalLink.njk" import externalLink %}
+
+{% set heading = 'Contact the court' %}
+
+{% block content %}
+
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <p>{{ t('If you are under 18 you need to email or phone the court. Claimants cannot submit a claim against someone under 18 so you may be able to have the claim stopped.') }}</p>
+
+      <p>
+        <strong>{{ t('Money Claim Online helpdesk') }}</strong><br/>
```

Comment:
We have few support macros, please check if we can reuse any of them.

Reply:
Clarifying with Pete as the contact details are a little different, but great idea! I didn't know about those widgets.

Reply:
Marc confirmed these details should be the same as is already present on the mcol eligibility page (and therefore in the macro)

Reply:
Thanks 👍